docs/pages/kubernetes-access/controls.mdx

```diff
@@ -487,19 +487,75 @@ spec:
+    resource corresponds to the name and namespace of a service account.
+  - `kube_node`: Controls access to nodes. The `name` field of a `kube_node`
+    resource corresponds to the name of a node.
+  - `persistentvolume`: Controls access to persistent volumes. The `name` field of a `persistentvolume`
+    resource corresponds to the name of a persistent volume.
+  - `persistentvolumeclaim`: Controls access to persistent volume claims. The `name` and `namespace` fields of a `persistentvolumeclaim`
+    resource corresponds to the name and namespace of a persistent volume claim.
+  - `deployment`: Controls access to deployments. The `name` and `namespace` fields of a `deployment`
+    resource corresponds to the name and namespace of a deployment.
+  - `replicaset`: Controls access to replica sets. The `name` and `namespace` fields of a `replicaset`
+    resource corresponds to the name and namespace of a replica set.
+  - `statefulset`: Controls access to stateful sets. The `name` and `namespace` fields of a `statefulset`
+    resource corresponds to the name and namespace of a stateful set.
+  - `daemonset`: Controls access to daemon sets. The `name` and `namespace` fields of a `daemonset`
+    resource corresponds to the name and namespace of a daemon set.
+  - `clusterrole`: Controls access to cluster roles. The `name` field of a `clusterrole`
+    resource corresponds to the name of a cluster role.
+  - `kube_role`: Controls access to roles. The `name` and `namespace` fields of a `kube_role`
+    resource corresponds to the name and namespace of a role.
+  - `clusterrolebinding`: Controls access to cluster role bindings. The `name` field of a `clusterrolebinding`
+    resource corresponds to the name of a cluster role binding.
+  - `rolebinding`: Controls access to role bindings. The `name` and `namespace` fields of a `rolebinding`
+    resource corresponds to the name and namespace of a role binding.
+  - `cronjob`: Controls access to cron jobs. The `name` and `namespace` fields of a `cronjob`
+    resource corresponds to the name and namespace of a cron job.
+  - `job`: Controls access to jobs. The `name` and `namespace` fields of a `job`
+    resource corresponds to the name and namespace of a job.
+  - `certificatesigningrequest`: Controls access to certificate signing requests. The `name` field of a `certificatesigningrequest`
+    resource corresponds to the name of a certificate signing request.
+  - `ingress`: Controls access to ingresses. The `name` and `namespace` fields of a `ingress`
+    resource corresponds to the name and namespace of a ingress.
 - `namespace`: The Kubernetes namespace in which to allow access to a resource.
   In the `kube-access` role, we are allowing access to a pod in the
   `production` namespace.
 - `name`: The name of the pod to allow access to. In `kube-access`, this is
   the `webapp` pod.
+- `verbs`: The operations to allow on the resource. Currently, Teleport supports:
```

Comment:
I would change this to a table as well

Reply:
236fbd2b919631466e288b1f3a408e18dce5b000